Formula/kube-linter.rb

```diff
@@ -16,8 +16,9 @@ class KubeLinter < Formula
   depends_on "go" => :build
 
   def install
-    system "make", "build"
-    bin.install ".gobin/kube-linter"
+    ENV["CGO_ENABLED"] = "0"
+    ldflags = "-s -w -X github.com/stackrox/kube-linter.VERSION=#{version}"
+    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/kube-linter"
```

Comment:
We can probably use a tarball instead of a Git clone now that we're doing `go build` instead of `make`.